docker-compose-cass-es.yml

```diff
@@ -29,7 +29,7 @@ services:
       - elasticsearch
     environment:
       - CASSANDRA_SEEDS=cassandra
-      - DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development_es.yaml
+      - DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development_cass.yaml
```

Comment:
is cass-es and cass the same now?

Reply:
Yes, and it almost empty. Server computes good defaults for all advanced visibility dynamic config settings based on static config.